DependencyInjection/Configuration.php

```diff
@@ -19,6 +19,7 @@ class Configuration implements ConfigurationInterface
 {
     /**
      * {@inheritdoc}
+     * @return TreeBuilder
```

Comment:
wouldn't type hint in method declaration be better than annotation?

Reply:
I agree it would be clearer, but as a quick fix for the deprecation warnings, the annotation is less intrusive. I the method does not return the requested type for some reason, it will fail which does not happen with only an annotation.